lib/livebook_web/live/settings_live.ex

```diff
@@ -124,6 +140,50 @@ defmodule LivebookWeb.SettingsLive do
     """
   end
 
+  defp autosave_path_select(assigns) do
+    ~H"""
+    <div class="flex items-center justify-between p-4">
+      <div class={"w-full #{if @state.dialog_opened?, do: "h-52", else: "h-8"}"}>
+        <%= if @state.dialog_opened? do %>
+          <.live_component module={LivebookWeb.FileSelectComponent}
+            id="autosave-path-component"
+            file={@state.file}
+            extnames={[]}
+            running_files={[]}
+            submit_event={:init}
```

Comment:
This is sent when hitting enter in the path input, we need to handle it the same way as "Save" :)